Sources/PointFree/Routes.swift

```diff
@@ -7,142 +8,123 @@ import Prelude
 import Styleguide
 
 public let siteMiddleware: Middleware<StatusLineOpen, ResponseEnded, Prelude.Unit, Data?> =
-  requireHerokuHttps
-    <<< redirectUnrelatedDomains
+  requireHerokuHttps(allowedInsecureHosts: allowedInsecureHosts)
+    <<< redirectUnrelatedHosts(allowedHosts: allowedHosts, canonicalHost: canonicalHosts)
     <<< route(router: PointFree.router)
+    <<< protectRoutes
     <| render(conn:)
+    >>> perform
```

Comment:
this is added cause I made `render(conn:)` return an `IO` instead of performing in there